src/azure-cli/azure/cli/command_modules/role/tests/latest/test_role.py

```diff
@@ -256,7 +256,7 @@ def test_role_definition_scenario(self):
 class RoleAssignmentScenarioTest(RoleScenarioTestBase):
 
     def __init__(self, *arg, **kwargs):
-        super().__init__(*arg, random_config_dir=True, **kwargs)
+        super().__init__(*arg, **kwargs)
```

Comment:
`random_config_dir=True` needs to be removed according to https://github.com/Azure/azure-cli/pull/26577#issuecomment-1677237581

Reply:
My preferred solution is to merge #26475.
This is a good chance to test it works. It only affect test, even if it contains bugs, it will not affect user.